packages/components/src/Form/Inputs/InputFilters/InputFilters.tsx

```diff
@@ -119,102 +119,123 @@ const InputFiltersLayout: FC<InputFiltersProps> = ({
+            newFilters[filterIndex] = updateFilter
 
-          onChange(newFilters)
-        }
+            onChange(newFilters)
+          }
 
-        const closeInputFiltersChipEditor = () => {
-          setFieldEditing(undefined)
-        }
+          const closeInputFiltersChipEditor = () => {
+            setFieldEditing(undefined)
+          }
 
-        const filterToken = filter.value ? (
-          <InputFiltersChip
-            filter={filter}
-            key={i}
-            onClick={editFilter}
-            onDelete={handleDelete}
+          const filterToken = filter.value ? (
+            <InputFiltersChip
+              filter={filter}
+              key={i}
+              onClick={editFilter}
+              onDelete={handleDelete}
+            />
+          ) : (
+            <Text fontSize="small">{filter?.label || filter.field}</Text>
+          )
+          return filter.field === fieldEditing ? (
+            <Popover
+              key={i}
+              isOpen={fieldEditing !== undefined}
+              setOpen={closeInputFiltersChipEditor}
+              content={
+                <InputFiltersChipEditor
+                  value={filter.value}
+                  onChange={setFieldEditingValue}
+                  options={filter.options}
+                />
+              }
+            >
+              {filterToken}
+            </Popover>
+          ) : (
+            filterToken
+          )
+        })}
+        {!fieldEditing && (
+          <Select
+            autoResize
+            openOnFocus
+            indicator={false}
+            onFilter={setFilterTerm}
+            isFilterable
+            onChange={handleFilterLookupChange}
+            options={options}
+            placeholder="Filter List"
+            ref={inputRef}
           />
-        ) : (
-          <Text fontSize="small">{filter?.label || filter.field}</Text>
-        )
-        return filter.field === fieldEditing ? (
-          <Popover
-            key={i}
-            isOpen={fieldEditing !== undefined}
-            setOpen={closeInputFiltersChipEditor}
-            content={
-              <InputFiltersChipEditor
-                value={filter.value}
-                onChange={setFieldEditingValue}
-                options={filter.options}
-              />
-            }
-          >
-            {filterToken}
-          </Popover>
-        ) : (
-          filterToken
-        )
-      })}
-
-      {!fieldEditing && (
-        <Select
-          autoResize
-          openOnFocus
-          indicator={false}
-          onFilter={setFilterTerm}
-          isFilterable
-          onChange={handleFilterLookupChange}
-          options={options}
-          placeholder="Filter List"
-          ref={inputRef}
-        />
-      )}
+        )}
+      </ChipWrapper>
       {isClearable && (
         <IconButton
           icon="Close"
           label="Clear Filters"
+          ml="auto"
           mr="xxsmall"
+          mt="xxsmall"
```

Comment:
Slight change to the margin here, and also add `mt="xxxsmall"` to the filter `Icon` above.
```suggestion
          mt="xxxsmall"
```